SQL/0000-00-00-schema.sql

```diff
@@ -1514,11 +1514,19 @@ CREATE TABLE `SNP` (
   `FunctionPrediction` enum('exonic','ncRNAexonic','splicing','UTR3','UTR5') DEFAULT NULL,
   `Damaging` enum('D','NA') DEFAULT NULL,
   `ExonicFunction` enum('nonsynonymous','unknown') DEFAULT NULL,
-  `GenomeLocID` bigint(20) DEFAULT NULL,
+  `Chromosome` varchar(255) DEFAULT NULL,
+  `Strand` varchar(255) DEFAULT NULL,
+  `EndLoc` int(11) DEFAULT NULL,
+  `Size` int(11) DEFAULT NULL,
```

Comment:
On the contrary, it allow it by removing 
```LEFT JOIN gene ON (gene.GenomeLocID = [CNV|SNP].GenomeLocID)```
Because a gene is never of size 1 (SNP) and has almost never the same coordinate than a CNV this filter would never have worked before except if genes where added accordin to CNV coordinates.